.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: go
+dist: trusty
+
+go:
+  - 1.7
+
+install:
+  - go get github.com/Sirupsen/logrus
```

Comment:
Hehe, so'll have a clash of Prometheus project standards and DO standards?
Would love to see an issue about the desired changes.

On Tue, Sep 13, 2016 at 10:28 PM Carlos Amedee <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/digitalocean/bind_exporter/pull/15#discussion_r78677898>
> :
>
> > @@ -0,0 +1,14 @@
> > +language: go
> > +dist: trusty
> > +
> > +go:
> > +  - 1.7
> > +
> > +install:
> > +  - go get github.com/Sirupsen/logrus
>
> We prefer vendoring. This pr is just to get some simple testing put in
> place. We will work on getting this project more aligned with the our
> general standard practices.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/digitalocean/bind_exporter/pull/15/files/17a9d8825a0f226b66398c3fb18cc87489879b68#r78677898>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAANaBHdYxIkRPNaD1pB8NkRd4uE030Cks5qp1vYgaJpZM4J8VOW>
> .
>